tests/metadata-ext.rs

```diff
@@ -83,3 +83,55 @@ fn test_metadata_ext() {
         );
     }
 }
+
+#[test]
+fn test_metadata_ext_created() {
```

Comment:
Is this test in the right / a reasonable place? I wasn't sure.

Reply:
Ah, good question. metadata-ext.rs is for `MetadataExt` APIs, but the `created()` field is a regular `Metadata` API. This can go in `fs_additional.rs`.

Reply:
Updated.